types/insane/README.md

```diff
@@ -0,0 +1,11 @@
+# Installation
```

Comment:
You can remove this file; it'll be generated automatically with these contents if it's not present

Reply:
Done